lib/graphql.ex

```diff
@@ -180,4 +184,23 @@ defmodule BridgeEx.Graphql do
       end
     end
   end
+
+  defmacro preload_query_from_file(name, opts \\ []) when is_atom(name) do
+    path = Keyword.get(opts, :path, "#{name}.graphql")
+
+    quote generated: true do
+      path = Path.join(__DIR__, unquote(path))
```

Comment:
I think this could cause problems if the user provides a full path

Reply:
Yes, I can't find a reason to support absolute paths here, so for now I would keep it simple.
I'll document this assumption.